lib/main_ltp.pm

```diff
@@ -41,6 +42,10 @@ sub load_kernel_tests {
         if (get_var('FLAVOR', '') =~ /Incidents-Kernel/) {
             loadtest_kernel 'update_kernel';
         }
+        if (is_alp) {
```

Comment:
I wonder if this new code should be put above `install_kotd`. Although it's unlikely these modules will be used for Alp, we never now.

Reply:
Moved up, but `install_kotd` will have to be updated as well for ALP, but this is not in the scope now.